### ISSUE TITLE
Twitter widgets probably broken

### ISSUE THREAD

**Issue by knirirr** [FAIRsharing/Sansone-Group-Website#16]
If I look at https://datareadiness.eng.ox.ac.uk I see all the Twitter widgets looking like the one below:

![Screenshot from 2023-10-03 14-15-03](https://github.com/FAIRsharing/Sansone-Group-Website/assets/3583375/4b810804-2356-45e3-99dd-505537d17e07)

@SusannaSansone , @allysonlister , do you also see that? I expect that Twitter is simply broken; here's what I get when looking at their own documentation on how to set up these feeds:

![Screenshot from 2023-10-03 14-19-53](https://github.com/FAIRsharing/Sansone-Group-Website/assets/3583375/f5d56911-e61d-4956-8ca8-33e4189d5675)

I could remove these. If you like, I could try to add Mastodon feeds instead. If so, which ones?

Of course, it could be an issue with my browsers (I have tried three), so if you can see the feed then there's nothing to do and this ticket may be closed.

**Comment by allysonlister** [comment]
I've seen this in my wordpress blog, so I think it's something that's happening to everyone?



--

Dr. Allyson Lister

FAIRsharing Coordinator - Content & Community


Oxford e-Research Centre, University of Oxford

Data Readiness Group: https://datareadiness.eng.ox.ac.uk/

FAIRsharing Community Champions: https://fairsharing.org/community_champions


ORCiD: 0000-0002-7702-4495
@allysonlister

Please note that I work part time. If the matter is urgent, you may wish to email ***@***.******@***.***> instead.


For more on FAIRsharing, please see https://fairsharing.org

________________________________
From: knirirr ***@***.***>
Sent: 03 October 2023 14:20
To: FAIRsharing/Sansone-Group-Website ***@***.***>
Cc: Allyson Lister ***@***.***>; Mention ***@***.***>
Subject: [FAIRsharing/Sansone-Group-Website] Twitter widgets probably broken (Issue #16)


If I look at https://datareadiness.eng.ox.ac.uk I see all the Twitter widgets looking like the one below:

[Screenshot from 2023-10-03 14-15-03]<https://user-images.githubusercontent.com/3583375/272274388-4b810804-2356-45e3-99dd-505537d17e07.png>

@SusannaSansone<https://github.com/SusannaSansone> , @allysonlister<https://github.com/allysonlister> , do you also see that? I expect that Twitter is simply broken; here's what I get when looking at their own documentation on how to set up these feeds:

[Screenshot from 2023-10-03 14-19-53]<https://user-images.githubusercontent.com/3583375/272275470-f5d56911-e61d-4956-8ca8-33e4189d5675.png>

I could remove these. If you like, I could try to add Mastodon feeds instead. If so, which ones?

**Comment by allysonlister** [comment]
most frustrating - thanks for letting us know



--

Dr. Allyson Lister

FAIRsharing Coordinator - Content & Community


Oxford e-Research Centre, University of Oxford

Data Readiness Group: https://datareadiness.eng.ox.ac.uk/

FAIRsharing Community Champions: https://fairsharing.org/community_champions


ORCiD: 0000-0002-7702-4495
@allysonlister

Please note that I work part time. If the matter is urgent, you may wish to email ***@***.******@***.***> instead.


For more on FAIRsharing, please see https://fairsharing.org

________________________________

**Comment by knirirr** [comment]
For the moment I have hidden the Twitter timelines as they're even more broken now than when this ticket was opened.
I have added the FAIRsharing Mastodon feed in the only place I could fit it.
@allysonlister presumably that's the only relevant Mastodon feed we have?

**Comment by allysonlister** [comment]
Yes I think that's all we'd want to include in the group - I don't think Mastodon feeds from me personally are that useful, and while there are others from other infrastructures, let's keep it simple. thanks!
